functions/system.bash

```diff
@@ -348,10 +348,10 @@ Finally, all common serial ports can be made accessible to the openHAB java virt
 
   if [[ $selection == *"3"* ]]; then
     cond_echo "Adding serial ports to openHAB java virtual machine in /etc/default/openhab2"
-    sed -i 's#^EXTRA_JAVA_OPTS=.*#EXTRA_JAVA_OPTS="-Xms250m -Xmx350m -Dgnu.io.rxtx.SerialPorts=/dev/ttyUSB0:/dev/ttyS0:/dev/ttyS2:/dev/ttyACM0:/dev/ttyAMA0"#g' /etc/default/openhab2
+    sed -i 's#^EXTRA_JAVA_OPTS=.*#EXTRA_JAVA_OPTS="-Xms192m -Xmx320m -Dgnu.io.rxtx.SerialPorts=/dev/ttyUSB0:/dev/ttyS0:/dev/ttyS2:/dev/ttyACM0:/dev/ttyAMA0"#g' /etc/default/openhab2
```

Comment:
Though the code is in there for a while,I do not like the way we do this here: Effectively removing all users settings and replacing everything by the default string. Grepping for the -X values is not a good way anyway if someone changed the file... just another reported issue...

Can't we just remove the -Dgnu... part of the line (just in case it is already there) and then add "[space]-Dgnu..." to the option? 
`sed  -i "/^[[:space:]]*EXTRA_JAVA_OPTS/s/-Dgnu.io.rxtx.SerialPorts=[^ \"]*//g" /etc/default/openhab2`
followed by
`sed -i "/^[[:space:]]*EXTRA_JAVA_OPTS/s#EXTRA_JAVA_OPTS[[:space:]]*=[[:space:]]*\"#EXTRA_JAVA_OPTS=\"-Dgnu.io.rxtx.SerialPorts=/dev/ttyUSB0:/dev/ttyS0:/dev/ttyS2:/dev/ttyACM0:/dev/ttyAMA0 #g" /etc/default/openhab2`

Reply:
Agree I dislike it, too. See also #522, #532 .
That should be undergoing a major overhaul to have separate env variables for user defined additions, HW related and openHABian-dictated values to combine into one Java args strings. 
But I don't want to do that before CI is ready as I'm afraid of side effects we don't think of now.

Reply:
ok